index.html

```diff
@@ -29,13 +29,140 @@ <h1 class="title">I'm Federica Glad to see you! </h1>
+                    <li class="list-item"><button disabled class= " btn btn-tags"> javaScript</button></li>
+                </ul>
+                <div class="action">
+                    <button disabled class="btn normal-b"> See project</button>
+                </div>
+            </div>
+        </div>
+        <div class="works multipost-uno">
+            <img src="img/SnapshootPortfolio2.png" class="snapshoot">
+            <div class="left-block">
+                <div class="primary-text-block">
+                    <h2 class="project-title">Multi-Post Stories</h2>
+                    <div class="frame-2">
+                        <h6 class="client">Canopy</h6>
+                        <img src="img/Counter.png" class="counter">
+                        <p class= "client-role">Back end dev</p>
+                        <img src="img/Counter.png" class="counter">
+                        <p class="client-year">2015</p>
+                    </div>
+                </div>
+                <p class= "primary-text multi">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                <ul class="tags-block">
+                    <li class="list-item"><button disabled class= "btn btn-tags"> html</button></li>
+                    <li class="list-item"><button disabled class= "btn btn-tags"> css</button></li>
+                    <li class="list-item"><button disabled class= " btn btn-tags"> javaScript</button></li>
+                </ul>
+                <div class="action">
+                    <button disabled class="btn normal-b"> See project</button>
+                </div>
+            </div>
+        </div>
+        <div class="works">
+            <img src="img/SnapshootPortfolio3.png" class="snapshoot">
+            <div class="left-block">
+                <div class="primary-text-block">
+                    <h2 class="project-title">Tonic</h2>
+                    <div class="frame-2">
+                        <h6 class="client">Canopy</h6>
+                        <img src="img/Counter.png" class="counter">
+                        <p class= "client-role">Back end dev</p>
+                        <img src="img/Counter.png" class="counter">
+                        <p class="client-year">2015</p>
+                    </div>
+                </div>
+                <p class= "primary-text">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                <ul class="tags-block">
+                    <li class="list-item"><button disabled class= "btn btn-tags"> html</button></li>
+                    <li class="list-item"><button disabled class= "btn btn-tags"> css</button></li>
+                    <li class="list-item"><button disabled class= " btn btn-tags"> javaScript</button></li>
+                </ul>
+                <div class="action">
+                    <button disabled class="btn normal-b"> See project</button>
+                </div>
```

Comment:
- [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
![image](https://user-images.githubusercontent.com/92303891/219876301-09d640e2-026a-441f-ad3f-748112f3e233.png)